crates/rpc/src/handlers/blob_sidecar.rs

```diff
@@ -0,0 +1,72 @@
+use actix_web::{
+    HttpResponse, Responder, get,
+    web::{Data, Json, Path},
+};
+use ream_consensus::blob_sidecar::{BlobIdentifier, BlobSidecar};
+use ream_storage::{db::ReamDB, tables::Table};
+use tracing::error;
+use tree_hash::TreeHash;
+
+use crate::{
+    handlers::block::get_beacon_block_from_id,
+    types::{errors::ApiError, id::ID, query::BlobSidecarQuery, response::BeaconVersionedResponse},
+};
+
+#[get("/beacon/blob_sidecars/{block_id}")]
+pub async fn get_blob_sidecars(
+    db: Data<ReamDB>,
+    block_id: Path<ID>,
+    query: Json<BlobSidecarQuery>,
+) -> Result<impl Responder, ApiError> {
+    let beacon_block = get_beacon_block_from_id(block_id.into_inner(), &db).await?;
+    let block_root = beacon_block.message.tree_hash_root();
+    let signed_block_header = beacon_block.compute_signed_block_header();
+
+    let indices = if let Some(indices) = &query.indices {
+        let max_index = beacon_block.message.body.blob_kzg_commitments.len() as u64;
+        for index in indices {
+            if index >= &max_index {
+                return Err(ApiError::BadRequest(format!(
+                    "Invalid blob index: {index}, max index is {}",
+                    max_index - 1
+                )));
+            }
+        }
+        indices.clone()
```

Comment:
```suggestion
    let indices = if let Some(indices) = query.indices {
        let max_index = beacon_block.message.body.blob_kzg_commitments.len() as u64;
        for index in indices {
            if index >= &max_index {
                return Err(ApiError::BadRequest(format!(
                    "Invalid blob index: {index}, max index is {}",
                    max_index - 1
                )));
            }
        }
        indices
```

alren't we able to remove this clone?